pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,356 @@
+
+`tags` can be defined in the [`components` object](../../reference/specification/v3.0.0-next-major-spec.12.md#componentsObject) of an AsyncAPI document, which enables the reusability of the tags. `tags` included in the `components` object can be re-used by using [reference objects](../../reference/specification/v3.0.0-next-major-spec.12.md#referenceObject).
+
+Additionally, you can define a list of tags you wanna use across the document under the `tags` object on the `info` level and you can reuse those tags under individual components such as `servers` or `channels` allowing you to group components logically.
+
+## `Tags` in AsyncAPI Document
+The `tags` object is a list of `tag` objects that can be referenced by [reference object](../../reference/specification/v3.0.0-next-major-spec.12.md#referenceObject).
+
+The `tags` object is a list of tags and individual `tag` objects contain some fields.
+
+The behaviour of tags inside the `tags` object in the AsyncAPI document can vary based on the context of the object it is being used in. For instance, you can use the `tags` object to have tags used consistently across the AsyncAPI document for logical grouping of components or you can use tags under individual components like on `servers` or `channels` for a specific purpose.
+
+Some of the fields that the `tags` object holds are:
+- `name`: The name of the tag.
+- `description`: A short description for the tag.
+- `externalDocs`: Additional external documentation for this tag.
+
+### `Tags` in `Info` Object
+When defined within the `tags` property of the info object, tags provide a high-level categorization and classification for the entire AsyncAPI document. These tags that are defined under the `info` object give a global context to the entire application, representing overarching themes or functional areas within the event-driven system. They serve as a way to group objects such as channels or servers based on their broader significance, providing a holistic understanding of the application's structure.
+
+Here's a visual representation of the `tags` object inside an `info` object in an AsyncAPI document:
+```mermaid
+graph LR
+  A[asyncapi] --> B[info]
+  B --> C[title]
+  B --> D[version]
+  B --> E[description]
+  B --> F[tags]
+  F --> G[name]
+  F --> I[description]
+  F --> K[externalDocs]
+
+  style A fill:#47BCEE,stroke:#000;
+  style B fill:#47BCEE,stroke:#000;
+  style F fill:#47BCEE,stroke:#000;
+  style I fill:#47BCEE,stroke:#000;
+  style K fill:#47BCEE,stroke:#000;
+  style G fill:#47BCEE,stroke:#000;
+```
+
+Below is an example of the `tags` object inside the `info` object in an AsyncAPI document:
+```yaml
+asyncapi: 3.0.0
+info:
+  title: AsyncAPI Documentation
+  version: 1.0.0
+  description: |
+    This AsyncAPI document provides an overview
+    of the event-driven system.
+  tags:
+    - name: Applications
+      description: All applications related topics. 
+      url: https://applications.example.com/docs
```

Comment:
I don't think it is a valid AsyncAPI document. Please always check with studio or other tools

you should have `externalDocs` first, with `description` and `url`

valid document could be:
```yml
asyncapi: 3.0.0
info:
  title: AsyncAPI Documentation
  version: 1.0.0
  description: |
    This AsyncAPI document provides an overview
    of the event-driven system.
  tags:
    - name: Applications
      description: All applications related topics. 
      externalDocs:
        description: More info about applications
        url: https://applications.example.com/docs
    - name: Time
      description: All time related topics.
      externalDocs:
        description: More info about time
        url: https://time.example.com/docs
```

Reply:
My bad, I was little confused on how to add the external docs component. Will make the changes throughout the document and validate using studio before updating